Add plonecli/bobtemplates.plone compatible markers and configuration (#63)
plone/cookieplone-templates: backend_addon/{{ cookiecutter.__folder_name }}/README.md

```shell
make create_site

> 

Reviewer: ```suggestion
```

```

> ## Add features using plonecli/bobtemplates.plone

Reviewer: ```suggestion
## Add features using `plonecli` or `bobtemplates.plone`
```

> This package provides plonecli/bobtemplates.plone compatible entrypoints to add all kind of subtemplates provided by them.

Reviewer: I can't figure out what you mean. Maybe this?

Also adding links for reference and convenience.

```suggestion
This package provides entry points that are compatible with [`plonecli`](https://github.com/plone/plonecli) and [`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone).
You can use the entry points to add all kinds of subtemplates.
```

Author: No, that's not the meaning. 

With the changes I added (the `<!-- extra stuff goes here -->` strings), if you use plonecli to add behaviors, control panels, upgrade steps, or whatever, they will be added and enabled correctly.

So these markers act as hooks where plonecli's subtemplates enter their content.

Reviewer 2: It'd be helpful to give an example or 2 of things that can be added using bobtemplates.plone, and to link to docs for how to use these tools.

Reviewer: Suggestion updated.

> To use them, you need to use bobtemplates.plone version > X.X.X and run your command as follows:

Reviewer: ```suggestion
To use them, you need to use `bobtemplates.plone` version X.X.X or later, and run the following command.
```

Author: Here we will need to set the version number of the new bobtemplate.plone package :)

```shell
plonecli add -b .mrbob.ini content_type
```

> This way we are passing the `.mrbob.ini` configuration file to plonecli to set some configuration variables needed to properly run the subtemplates.

Reviewer: ```suggestion
The command passes the `.mrbob.ini` configuration file to `plonecli` to set some configuration variables which are needed to properly run the subtemplates.
```

make create_site
```

## Add features using `plonecli` or `bobtemplates.plone`

This package provides markers as strings (`<!-- extra stuff goes here -->`) that are compatible with [`plonecli`](https://github.com/plone/plonecli) and [`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone).
These markers act as hooks to add all kinds of subtemplates, including behaviors, control panels, upgrade steps, or other subtemplates from `plonecli`.

> To use them, you need to use `bobtemplates.plone` version X.X.X or later, and run the following command.

Reviewer: We need to update the version number here. The current version of `bobtemplates.plone` is 6.3.3. Is this feature already in a release or a future one?

Author: No yet, there's a pending PR on bobtemplates.plone to be able to use these changes.

Reviewer 2: @erral Any update?

Author: The PR was merged, but the release is still missing.

make create_site
```

## Add features using `plonecli` or `bobtemplates.plone`

This package provides markers as strings (`<!-- extra stuff goes here -->`) that are compatible with [`plonecli`](https://github.com/plone/plonecli) and [`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone).
These markers act as hooks to add all kinds of subtemplates, including behaviors, control panels, upgrade steps, or other subtemplates from `plonecli`.

> To use them, you need to use `bobtemplates.plone` version 6.3.4 or later, and run the following command.

Reviewer 2: This assumes plonecli is already installed. How should the developer install it? Should these docs be updated to use the `make add` command?

Reviewer 2: Done in https://github.com/plone/cookieplone-templates/pull/63/commits/72a2c0866156ebf3fea7bdb2ecec9cfad8af1764

```shell
plonecli add -b .mrbob.ini content_type
```

The command passes the `.mrbob.ini` configuration file to `plonecli` to set some configuration variables which are needed to properly run the subtemplates.

For example, you can add a behavior to your package with the following command.

```shell
plonecli add -b .mrbob.ini behavior
```

You can add a control panel with the following command.

```shell
plonecli add -b .mrbob.ini controlpanel
```

```{seealso}
You can check the list of available subtemplates in the [`bobtemplates.plone` `README.md` file](https://github.com/plone/bobtemplates.plone/?tab=readme-ov-file#provided-subtemplates).

> See also the documentation of [Mockup and Patternslib](https://6.docs.plone.org/classic-ui/mockup.html) for how to build the UI toolkit for Classic UI.

Reviewer: I opened a PR that needs to be reviewed and merged before this is finalized. See:

https://github.com/plone/documentation/pull/1734

Reviewer: PR merged!
